src/plonkish/backend/halo2.rs

```diff
@@ -120,7 +121,7 @@ pub struct ChiquitoHalo2<F: PrimeField + From<u64>> {
     pub(crate) plonkish_ir: Circuit<F>,
 
     advice_columns: HashMap<UUID, Column<Advice>>,
-    fixed_columns: HashMap<UUID, Column<Fixed>>,
+    fixed_columns: HashMap<UUID, (Column<Fixed>, Option<TableColumn>)>,
```

Comment:
@alxkzmn perhaps can we do the distinction already before filling this struct, so instead of the option we would have two hashmaps?

Reply:
Or this complicates the code?

Reply:
or it seems that allocate_fixed_columns should return two hashmaps, one for fixed and another for table

Reply:
Yes, this is possible.